<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,10,2,4">
    <item>
     <layout class="QVBoxLayout" name="PhaseLayout"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="FieldLayout" stretch="2,1,2,1">
      <item>
       <widget class="QWidget" name="OpponentMainField" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
        border: 2px solid #555;
        border-radius: 10px;
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="ShowdownField" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
        border: 2px solid #555;
        border-radius: 10px;
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="PlayerMainField" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
        border: 2px solid #555;
        border-radius: 10px;
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="PlayerMainFieldLayout"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="HandWidget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
        border: 2px solid #555;
        border-radius: 10px;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="HandLayout"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="SideLayout" stretch="0,1,1,1,1,1,0">
      <item>
       <widget class="QLabel" name="OpponentLP">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="OpponentDeck" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
        border: 2px solid #555;
        border-radius: 10px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="OpponentGraveyard" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
        border: 2px solid #555;
        border-radius: 10px;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>150</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="PlayerGraveyard" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
        border: 2px solid #555;
        border-radius: 10px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="PlayerDeck" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
        border: 2px solid #555;
        border-radius: 10px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="DeckSize">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="PlayerLP">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="InfoLayout" stretch="1,1">
      <item>
       <widget class="QTabWidget" name="CardPreview">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="Image">
         <attribute name="title">
          <string>Image</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="ImagePreview">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Description">
         <attribute name="title">
          <string>Description</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="DescriptionPreview">
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LogPanel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
